worlds/ff6wc/Items.py

```diff
@@ -52,4 +52,44 @@ class ItemData(typing.NamedTuple):
     "Dragon Horn"
 ]
 
+# NOTE: replaced by item_nonrewards in __init__.py due to FF6WC 1.4.2 allowing custom item rewards
 okay_items = [item for item in items if item not in good_items or item == "Empty" or item == "ArchplgoItem"]
+
+stronger_items = [
```

Comment:
```suggestion
]

stronger_items = [
```
I think it's ok to delete this, unless you can think of a reason we might still need it.

Reply:
We can probably delete okay_items, I just left things in but added comments for history.